org.eclipse.wb.swt/src/org/eclipse/wb/internal/swt/model/property/editor/color/ColorPropertyEditor.java

```diff
@@ -217,10 +217,8 @@ private static String getColorFieldName(Expression idExpression) throws Exceptio
       }
     }
     // no field, unexpected
-    throw new IllegalArgumentException(MessageFormat.format(
-        ModelMessages.ColorPropertyEditor_wrongSwtColor,
-        idExpression,
-        id));
+    throw new IllegalArgumentException(

```

Comment:
Format changes and real changes should not be mixed. I like this reformat, please provide separate PR for this.